### DISCUSSION TITLE
Unable to download clear photo of my Student ID because my web camera is of poor quality

### DISCUSSION THREAD

**Discussion by bluetail14** [community/community#127387] [GitHub Education]
### Hi everyone,

I am trying to take a photo of my student ID to join GitHub Education. 
The only option I can see is using my web camera, which is attached to my notebook with a usb cable. however, my camera is cheap and poor and the photo is blurry. it does not pass the verification check.
My smart phone would make a better photo, but I do not see how can I take a photo of my ID with my phone instead? when I try to connect to my phone via a usb cable, it says 'Error: no video devices detected'.

![upload_photo_github](https://github.com/community/community/assets/47062263/9e9b54e9-bdaa-468e-8656-0aab1ad27a93)

How can I overcome this problem?  I have tried submitting a ticket but seems no longer possible as they do not process applications manually anymore.  I have successfully verified my student email and my university.

**Comment by davevad93** [comment]
Hi @bluetail14 ,

When I applied for my benefits I was able to upload my documentation by clicking on `Upload` button. If there's no upload button, you should try submitting your application via smartphone so you can take a much better photo of your ID.

**Reply by LeoHenriot** [reply]
We don't have "upload" button as you can see...
